test/HubPoolClient.DepositToDestinationToken.ts

```diff
@@ -22,9 +23,37 @@ import {
 
 let hubPool: Contract, lpTokenFactory: Contract, mockAdapter: Contract;
 let owner: SignerWithAddress;
-let hubPoolClient: HubPoolClient;
+let hubPoolClient: MockHubPoolClient;
 
 describe("HubPoolClient: Deposit to Destination Token", function () {
```

Comment:
All of the new HubPoolClient functions are unit tested rigorously in this file, PTAL